README.md

```diff
@@ -75,6 +77,14 @@ You can install or update the SDK using the Ant Build Tool by following these st
     ant deployWatson
     ```
 
+    By default, this will deploy all of the classes. To specify a service, use the `-Dservice` flag. Here is an example of just deploying the Conversation service:
+
+    ```bash
+    ant deployWatson -Dservice=conversation
+    ```
+
+    To deploy multiple services, just run the command again with the new service flag.
```

Comment:
Include list of valid services available for deployment

* conversation
* discovery
* naturallanguageclassifier
* personalityinsights
* visualrecognition
* naturallanguageunderstanding